scripts/run_backend_tests.py

```diff
@@ -493,14 +511,23 @@ def main(args=None):
             '%s errors, %s failures' % (total_errors, total_failures))
 
     if parsed_args.generate_coverage_report:
-        subprocess.check_call(['python', COVERAGE_PATH, 'combine'])
-        subprocess.check_call([
-            'python', COVERAGE_PATH, 'report',
-            '--omit="%s*","third_party/*","/usr/share/*"'
-            % common.OPPIA_TOOLS_DIR, '--show-missing'])
+        subprocess.check_call(['python', '-m', 'coverage', 'combine'])
```

Comment:
Maybe use `common.call_cmd`?

Reply:
That doesn't let the error surface out as-is, so I'm going to keep this here.